src/JBrowse/Store/TabixIndexedFile.js

```diff
@@ -40,6 +40,25 @@ return declare( null, {
         }, errorCallback);
     },
 
+    countLines: function( ref, min, max, itemCallback, finishCallback, errorCallback ) {
+        var thisB = this;
+        var args = Array.prototype.slice.call(arguments);
+        this.indexLoaded.then(function() {
+            thisB._count.apply( thisB, args );
+        }, errorCallback);
+    },
+
+    _count: function( ref, min, max, itemCallback, finishCallback, errorCallback ) {
```

Comment:
this function doesn't look right.  it is returning count synchronously, but it also calls an error callback.

and it's returning a count of the blocks in the range, not the lines.  what's up here?

Reply:
good idea.  I removed it as they weren't actually used.